### ISSUE TITLE
async function play() SnytaxError: Unexpected token function

### ISSUE THREAD

**Issue by critzlez** [TheDemise/discord.js-musicbot-addon#7]
After (I assume) your latest update, I can't start my bot with this addon, as it gives me this error. is there a way to fix it?
```	
    async function play(msg, suffix) {
          ^^^^^^^^

SyntaxError: Unexpected token function
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/critzlez/Documents/bot/bot.js:8:15)
```

**Comment by TheDemise** [comment]
I'm fairly sure I've had the play function asynchronous for quite some time, but I could be wrong. I don't really see _why_ it would be a problem for you however. Removing the `async` should fix that, but it may cause (very, _very_ rare) issues with the play command. It's unlikely they'll happen either way but having it asynchronous made sure some errors wouldn't happen.

**Comment by critzlez** [comment]
Before, I found a site that told me to update to at least node 8 for the async function to work. I did so, however most of my other modules in my discord bot breaks because of that. I removed that, and the warning that states I need node 8. It works now. Thanks!

However, I think you might have to address this problem though. This module requires me to use node 8, however the LTS version is 6, and more people use LTS instead of version 8.
